shpc/tests/test_views.py

```diff
@@ -76,7 +76,7 @@ def test_views(tmp_path, module_sys, module_file, container_tech, remote):
     assert not view._config["view"]["system_modules"]
 
     # Now install to it via the client
-    client.install("ghcr.io/autamus/emacs:27.2", view=view_name)
+    client.install("ghcr.io/autamus/emacs:27.2", extra_view=view_name)
```

Comment:
Perhaps just `view` here that is a kwarg, and that populates into a variable `extra_view`?